README.md

```diff
@@ -1,54 +1,60 @@
+> NOTES:
+> These images are not associated with System76! If you have issues, please understand they might be COSMIC related, OR they might be related to this image.
+> The COSMIC Desktop Environment is still PRE ALPHA. Do not daily drive this image on your main workstation unless you know what you're doing.
 
 Like Fedora? Want to try the latest from the work in progress Cosmic Desktop Environment? Want to help find bugs and/or contribute to Cosmic development, but don't want to work in a VM or install Pop!_OS? None of the above things but something else??!?
 
 Go ahead and try one of the ostree images I've created here!
 
 ### Quick Installation
 
-Install an rpm-ostree based desktop, like [Fedora Silverblue](https://fedoraproject.org/atomic-desktops/silverblue/) (aka atomic desktops)
+Install a Fedora Atomic Desktop, best [Fedora Silverblue](https://fedoraproject.org/atomic-desktops/silverblue/).
 
 #### Warning
-This image requires disabling SELinux. This is NOT recommended for production and is a temporary situation until this work is finished in upstream Fedora. 
+This image requires disabling SELinux. **This is NOT recommended for production** and is a temporary situation until this work is finished in upstream Fedora. 
 
-Edit `/etc/selinux/config`, set from `enforcing` to `permissive`
+    sudo setenforce 0 && getenforce
 
-> Note: Rebase to an unsigned image before rebasing to the signed cosmic images below!
+You can view the selinux config in `/etc/selinux/config`
 
-```
-$ sudo rpm-ostree rebase ostree-unverified-registry:ghcr.io/ublue-os/cosmic-base:40-amd64
-```
+#### Variants
+- `cosmic-base`: Just the COSMIC Desktop
+- `cosmic-silverblue`: Recommended, Fedora Silverblue with COSMIC Desktop added
+- `cosmic-kinoite`: Fedora Kinoite with COSMIC Desktop addded
 
-Run this command to rebase onto the signed image:
+> NOTE: Rebase to the signed image, the unsigned image is only needed during the transition
 
-```
-$ sudo rpm-ostree rebase ostree-image-signed:docker://ghcr.io/ublue-os/cosmic-base:40-amd64
-```
+Rebase to the temporary unsigned image
 
-Or this command if you're running an arm device:
+    rpm-ostree rebase --reboot ostree-unverified-registry:ghcr.io/ublue-os/VARIANT:40-amd64
 
-```
-$ sudo rpm-ostree rebase ostree-image-signed:docker://ghcr.io/ublue-os/cosmic-base:40-arm64
-```
+Rebase to the signed image
 
-Reboot
+    rpm-ostree rebase --reboot ostree-image-signed:docker://ghcr.io/ublue-os/VARIANT:40-amd64
 
-    systemctl reboot
+#### If you are on an ARM device
+> NOTE: Apple M-series Laptops are not supported by default Fedora ARM, use [Fedora Asahi Remix](https://asahilinux.org/fedora/)
+
+Rebase to the temporary unsigned image
+
+    rpm-ostree rebase --reboot ostree-unverified-registry:ghcr.io/ublue-os/VARIANT:40-arm64
+
+Rebase to the signed image
+
+    rpm-ostree rebase --reboot ostree-image-signed:docker://ghcr.io/ublue-os/VARIANT:40-arm64
```

Comment:
```suggestion
```

Reply:
We don't have arm images so we don't need this.

Reply:
lol this was already there, so I will remove the whole part